src/NLU.DevOps.ModelPerformance/Utilities.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace NLU.DevOps.ModelPerformance
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Text;
+    using NLU.DevOps.ModelPerformance;
+
+    /// <summary>
+    /// This class contains utils function for performance calculations
+    /// and printing
+    /// </summary>
+    public static class Utilities
```

Comment:
Can we give this a more descriptive name than Utilities?

Reply:
I changed it to NLUAccuracy

---
In reply to: [376177211](https://github.com/microsoft/NLU.DevOps/pull/250#discussion_r376177211) [](ancestors = 376177211)